diff --git a/node_modules/react-leaflet/lib/MapContainer.d.ts b/node_modules/react-leaflet/lib/MapContainer.d.ts
index 9cc061e..516087b 100644
--- a/node_modules/react-leaflet/lib/MapContainer.d.ts
+++ b/node_modules/react-leaflet/lib/MapContainer.d.ts
@@ -8,6 +8,6 @@ export interface MapContainerProps extends MapOptions {
     id?: string;
     placeholder?: ReactNode;
     style?: CSSProperties;
-    whenReady?: () => void;
+    whenReady?: (map: {target: LeafletMap}) => void;
 }
 export declare const MapContainer: React.ForwardRefExoticComponent<MapContainerProps & React.RefAttributes<LeafletMap>>;
